acceptance/tests/security/cve-2013-1654_sslv2_downgrade_agent.rb

```diff
@@ -72,9 +72,11 @@ def run
       begin
         Timeout.timeout(timeout) do
           loop do
+            # 6 is "Could not resolve host", which is happening transiently in CI, see QENG-3328
             # 7 is "Could not connect to host", which will happen before it's running
             # 28 is "Operation timeout", which could happen if the vm was running slowly
-            result = on(agent, "curl --tlsv1 -m1 -k https://#{agent}:#{port}", :acceptable_exit_codes => [0,7,28,35])
+            result = on(agent, "curl --tlsv1 -m1 -k https://#{agent}:#{port}", :acceptable_exit_codes => [0,6,7,28,35])
+            logger.debug( "Retrying curl after name resolution failure" ) if result.exit_code == 6
             break if result.exit_code == 0 or result.exit_code == 35
```

Comment:
I was thinking the idea would be to actually retry (up to 15 seconds) and see if the name resolution ends up successful.